internal/errors/code.go

```diff
@@ -61,6 +61,10 @@ const (
 	GenKey  Code = 304 // GenKey represents an error occurred during the underlying key generation process
 	GenCert Code = 305 // GenCert represents an error occurred during the underlying certificate generation process
 
+	// General system errors are reserved Codes 400-599 and align with http client and server error codes
+	Forbidden Code = 403 // Forbidden represents the operation is forbidden
```

Comment:
After this PR is merged, would you mind actually moving the changes in the files in this package just directly into `main`? I think it's broadly useful even now. Wouldn't even bother PRing it as it's already approved here (or PR it and just merge).

Reply:
Happy to.  (adding it to my TODO list)